.vscode/settings.json

```diff
@@ -3,30 +3,11 @@
   "python.linting.mypyEnabled": true,
   "files.trimTrailingWhitespace": true,
   "editor.formatOnSave": true,
-  "editor.detectIndentation": false,
+  "editor.detectIndentation": true,
   "editor.insertSpaces": true,
-  "editor.tabSize": 2,
-  "[python]": {
-    "editor.tabSize": 4,
-    "editor.rulers": [
-      72,
-      79,
-      119
-    ]
-  },
-  "[yaml]": {
-    "editor.tabSize": 4
-  },
-  "[rst]": {
-    "editor.rulers": [
-      79,
-      119
-    ]
-  },
-  "[markdown]": {
-    "editor.rulers": [
-      79,
-      119
-    ]
-  }
+  "editor.rulers": [
+    72,
+    79,
+    119
+  ]
```

Comment:
Was this intentionally committed?

Reply:
Ah yes, figured we could discuss it, but I think we can just have those
rulers everywhere.

And we I think letting VSCode automatically detect the indentation works
well enough...

Reply:
I personally found the 72 line a little annoying for `.md` and `.rst`...

Reply:
Would something like:

```js
  "editor.rulers": [
    79,
    119
  ],
"[python]": {
    "editor.rulers": [
      72,
      79,
      119
    ]
  },
```

be alright?